.github/ISSUE_TEMPLATE/report-compatibility.md

```diff
@@ -0,0 +1,35 @@
+---
+name: "📝 Make a Compatibility Report"
+about: Report Portmaster in/compatibility with Linux Distros, VPN Clients or general Software
+labels: compatibility
+---
+
+**Pre-Submit Checklist**:
+
+- Check applicable sources for existing issues
+  - [Linux Compatibility](https://docs.safing.io/portmaster/install/linux#compatibility)
+  - [VPN Compatibility](https://docs.safing.io/portmaster/install/status/vpn-compatibility)
+  - [Github Issues (all our repos)](https://github.com/issues?q=user%3Asafing)
```

Comment:
this is a stretch and surely not applicable - we will collect these only in the Portmaster repo, no?

Reply:
Going to go with the current repo, filtered by the respective issue tag.